monitoring/monitoring.go

```diff
@@ -52,12 +51,9 @@ const (
 	YearPeriod
 )
 
-const (
-	monitoringChannelSize = 64
-	// For optimization capacity should be equals numbers of measurement values
-	// 5 - RAM, CPU, UsedDisk, InTraffic, OutTraffic.
-	capacityAlertProcessorElements = 5
-)
+// For optimization capacity should be equals numbers of measurement values
```

Comment:
Change commit message, it doesn't add monitoring functionality. Should be: remove monitoring data channel or similar.

Reply:
I squashed into one comment